distribution/lib/Standard/Base/0.0.0-dev/src/Data/Array.enso

```diff
@@ -331,6 +331,28 @@ type Array
        Arguments:
        - function: A function that takes an element in the array and returns
          some transformation of that element.
+       - on_problems: Specifies how to handle any problems that arise in
+         `function`.
+
+       ? Problem Handling
+
+         The result of Errors thrown when executing the function depend on
+         `on_problems`:
+         - Problem_Behavior.Report_Error: The error is thrown.
+         - Problem_Behavior.Report_Warning: The result is `Nothing`, with the
+           error attached as a warning. Currently unimplemented.
+         - Problem_Behavior.Ignore: The result is `Nothing`, and the error is
+           ignored.
+
+       ! Error Conditions
+
```

Comment:
Not sure why this would be a separate section, IMHO it can be continuation of that previous Problem Handling section - as it is not introducing a separate concept but clarifying the one above.

Reply:
Done